keyboards/xbows/knight/knight.c

```diff
@@ -0,0 +1,20 @@
+#include "knight.h"
+void led_set_kb(uint8_t usb_led) {
```

Comment:
You can change all these to GPIO for example

```
if (IS_LED_ON(usb_led, USB_LED_NUM_LOCK)) {
    writePinHigh(D1);
} else {
    writePinLow(D1);
}

Reply:
changed, thanks!